dpl.gemspec

```diff
@@ -16,12 +16,12 @@ Gem::Specification.new do |s|
   s.require_path          = 'lib'
   s.required_ruby_version = '>= 1.9.3'
 
-  s.add_development_dependency 'rspec', '~> 3.0.0'
-  s.add_development_dependency 'rspec-its'
-  s.add_development_dependency 'rake'
-  s.add_development_dependency 'json', '1.8.1'
-  s.add_development_dependency 'tins', '~> 1.6.0', '>= 1.6.0'
-  s.add_development_dependency 'coveralls'
+  s.add_development_dependency 'rspec', '~> 3.0', '>= 3.0.0'
+  s.add_development_dependency 'rspec-its', '~> 1.2', '>= 1.2.0'
+  s.add_development_dependency 'rake', '~> 12.0', '>= 12.0.0'
+  s.add_development_dependency 'json', '~> 1.8', '>= 1.8.1'
```

Comment:
I don't think we can relax this condition. See https://github.com/travis-ci/dpl/issues/513 for some more info.

I don't know if how other proposed changes affect the overall fitness of gem specifications.